--- a/src/Section.php	(revision 73ecabce67254cdc0b28f4d2caddf515a5f1ab49)
+++ b/src/Section.php	(date 1613330963253)
@@ -71,7 +71,7 @@
     private function setDeclarations(array $declarations) : void
     {
         foreach ($declarations as $name => $value) {
-            $this->setDeclaration($name, $value);
+            $this->setDeclaration($name, is_string($value) ? strtolower($value) : $value);
         }
     }
